dashboards/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana dashboards
+
+We provide a dashboards for Grafana, leveraging the exporter.
+
+
+## SAP NetWeaver Details
+
+This dashboard shows the details of a single landscape.
+
+![SAP NetWeaver Details](screenshot-details.png)
+
+
+## Installation
+
+### RPM 
+
+On openSUSE and SUSE Linux Enterprise distributions, you can install the package via zypper in your Grafana host:
+```
+zypper in grafana-sap-netweaver-dashboards
+systemctl restart grafana-server
+```
+
+For the latest development version, please refer to the [development upstream project in OBS](https://build.opensuse.org/project/show/network:ha-clustering:sap-deployments:devel), which is automatically updated everytime we merge changes in this repository. 
+
+### Manual
+
+Copy the [provider configuration file](https://build.opensuse.org/package/view_file/network:ha-clustering:sap-deployments:devel/grafana-sap-providers/provider-sles4sap.yaml?expand=1) in `/etc/grafana/provisioning/dashboards` and then the JSON files inside `/var/lib/grafana/dashboards/sles4sap`.
+
+Once done, restart the Grafana server.
+
+### Grafana.com
```

Comment:
imho we should add the `grafana.com` link when they are avail to avoid that we forget to update doc

Reply:
I removed the section altogether